css-values/Overview.bs

```diff
@@ -1088,27 +1185,93 @@ Absolute lengths: the ''cm'', ''mm'', ''q'', ''in'', ''pt'', ''pc'', ''px'' unit
 		    <th>name
 		    <th>equivalence
 	<tbody dfn-type=value dfn-for="<length>">
+		<tr><th><dfn id="m">m</dfn>
+		    <td>meters
+		    <td>1m = 1000mm
+		<tr><th><dfn id="dm">dm</dfn>
+		    <td>decimeters
+		    <td>1dm = 100mm
 		<tr><th><dfn id="cm">cm</dfn>
 		    <td>centimeters
-		    <td>1cm = 96px/2.54
+		    <td>1cm = 10mm
 		<tr><th><dfn id="mm">mm</dfn>
 		    <td>millimeters
-		    <td>1mm = 1/10th of 1cm
+		    <td>1mm = 96px/25.4 
+		<tr><th><dfn id="mu">mu</dfn>
+			<!--'um'?-->
+		    <td>micrometers (&mu;m), "microns"
+		    <td>1mu = 1mm/1000
 		<tr><th><dfn id="q">Q</dfn>
-		    <td>quarter-millimeters
-		    <td>1q = 1/40th of 1cm
+		    <td>quarter-millimeters (&#x30ad;&#x30a5;)
+		    <td>1q = 1mm/4
+		<tr><th><dfn id="cc">cc</dfn>
+		    <td>ciceros
+		    <td>1cc = 12dd = 18q
+		<tr><th><dfn id="dd">dd</dfn>
+		    <td>didots
+		    <td>1dd = 3mm/8 = 1.5q
+		<tr><th><dfn id="thm">thm</dfn>
+			<!--tm, th, thb, tb, nn, ii, inm "metric inch"-->
+		    <td>thumbs
+		    <td>1thm = 25mm
+		<tr><th><dfn id="mdl">mdl</dfn>
+			<!--nn, mod, ftm "metric foot"-->
+		    <td>modules
+		    <td>1mdl = 300mm = 12thm
+		<tr><th><dfn id="dot">dot</dfn>
+		    <td>dots 
+		    	<!-- gcd(1q, 1dd, 1px, 1twip) => max(signed 32bit Int) = ca. 1.5 km -->
+		    <td>1dot = 1mm/1440
 		<tr><th><dfn id="in">in</dfn>
 		    <td>inches
-		    <td>1in = 2.54cm = 96px
+		    <td>1in = 25.4mm = 96px
 		<tr><th><dfn id="pc">pc</dfn>
 		    <td>picas
-		    <td>1pc = 1/6th of 1in
+		    <td>1pc = 1in/6 = 12pt = 16px
+		<tr><th><dfn id="sx">sx</dfn>
+			<!--"1st" would read "first" -->
+		    <td>sixteenth-inches
+		    <td>1sx = 1in/16 = 6px
+		<tr><th><dfn id="tx">tx</dfn>
+		    <td>thirtysecondth-inches
+		    <td>1tx = 1in/32 = 3px
 		<tr><th><dfn id="pt">pt</dfn>
 		    <td>points
-		    <td>1pt = 1/72th of 1in
```

Comment:
use notation valid inside `calc()` instead of a mix of English and code notation